proxies/live/apiproxy/policies/OAuthV2.VerifyAccessToken.xml

```diff
@@ -1,3 +1,4 @@
 <OAuthV2 async="false" continueOnError="false" enabled="true" name="OauthV2.VerifyAccessToken">
   <Operation>VerifyAccessToken</Operation>
+  <Scope>urn:nhsd:apim:app:level3:e-referrals-service-api urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api</Scope>
```

Comment:
Doesn't defining required scopes in this manner result in an and rather than an or? So detailing that a client's access token must have both the application restricted scope and the user restricted scope?